frontend/src/App.js

```diff
@@ -1,7 +1,5 @@
 import React, { Component } from 'react';
-import WelcomePanel from './components/panels/home_panel/WelcomePanel';
-import ProjectListPanel from './components/panels/project_list_panel/ProjectListPanel';
-import ProjectFiltersPanel from './components/panels/project_filters_panel/ProjectFiltersPanel';
+import {WelcomePanel, ProjectListPanel, ProjectFiltersPanel}  from './components/panels/index';
```

Comment:
Due to how node imports work you don't actually have to import from index. This could simply be `./components/panels;`

On another note and this is completely just nitpicking...be consistent with the spacing around brackets. Basically either have the space between the bracket or don't. It's up to you. On that note I'd setup an `.eslintrc` file so people can configure their IDE to pickup on the styling rules you define.